core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalWriter.java

```diff
@@ -83,7 +83,7 @@ public void write(JournalEntry entry) throws IOException, JournalClosedException
     if (mClosed) {
       throw new JournalClosedException("Cannot write to journal. Journal writer has been closed");
     }
-    Preconditions.checkState(entry.getAllFields().size() <= 1,
+    Preconditions.checkState(entry.getAllFields().size() <= 2,
```

Comment:
Can you help me understand the change to this precondition check?

Reply:
There is an `OpId` added to the journal entry so there's one additional field now. 

But it's good to elaborate in the comments.

Reply:
What is a waste?

Reply:
Consider the scenario where retries handled by multiple Alluxio masters. We need to mark each journal entry with a unique operationId. Or followers would have no idea which operation is applied or not.